recipes/pandas-profiling/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pandas-profiling" %}
+{% set version = "1.1.0" %}
+{% set md5 = "22f1c22418b5c07f371735e74440e631" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py3k]
```

Comment:
So I don't think this uses `ebooklib`, right? Just `pandas` and `matplotlib`.

Please see PR ( https://github.com/conda-forge/pandas-profiling-feedstock/pull/1 ) for enabling Windows builds.